src/nlp_sdg/pipelines/data_engineering/nodes.py

```diff
@@ -6,16 +6,125 @@
 import numpy as np
 import pandas as pd
 from pyspark.sql import DataFrame
+import texthero as hero
+from nltk.corpus import stopwords
+from nltk.corpus import stopwords, wordnet
+from nltk.stem import WordNetLemmatizer
 
 
-def dummy_node(data: DataFrame) -> DataFrame:
-    """Dummy node to read data
+def clean_agreement(data: pd.DataFrame) -> pd.DataFrame:
```

Comment:
@JayHansea, this is a general comment: can we please make use of an underscore for all functions that are local to this script. i.e. if the function is only used within nodes.py it should be named as follows: **_<name_of_function>**. Look at the **_lemmatize** function in this notebook.

This convention will help us know which **functions are nodes**, and which are just **helper-functions**